openerp/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -468,8 +468,14 @@ def test_60_x2many_domain(self):
         self.assertNotIn(message, discussion.important_messages)
 
         message.important = True
+        discussion.invalidate_cache()
```

Comment:
I had to add this line because somehow it seems like cache for models that have a computed domain do not update cache correctly. If you tell me how to fix that, I can leverage this PR to fix that too, but honestly it's not so much critical as the issue I'm addressing here.